### PR TITLE
Wqp 1270 part2 update summary station names

### PR DESCRIPTION
**Pull request by abriggs-usgs**
Before making a pull request
----------------------------

- [x] Make sure all tests run
- [x] Update the changelog appropriately

WQP-1270 Part 2, Update Summary Station Name
-----------
This pull request is a continuation of WQP-1270 part 1. Here the use of  'summarystation' is removed and replaced with 'summaryMonitoringLocation.'

After making a pull request
---------------------------
- [x] If appropriate, put the link to the PR in the JIRA ticket
- [x] Assign someone to review unless the change is trivial

### CODE REVIEW

File: src/main/resources/mybatis/dynamicStation.xml [hunk 1]
```diff
@@ -48,7 +48,7 @@
 
     <sql id="spatialNeeded">
         <if test="mimeType != null and (mimeType == 'geojson' or mimeType == 'json' or mimeType == 'kml' or mimeType == 'kmz')
-                  and dataProfile != null and (dataProfile == 'station' or dataProfile == 'simplestation' or dataProfile == 'summarystation')">
+                  and dataProfile != null and (dataProfile == 'station' or dataProfile == 'simplestation' or dataProfile == 'monitoringlocation')">
```
Comment (dsteinich) [inline src/main/resources/mybatis/dynamicStation.xml]: "monitoringlocation" might need to be both of the summary profiles for monitoring location. (Profile.SUMMARY_MONITORING_LOCATION & Profile.PERIOD_OF_RECORD values)

Comment (abriggs-usgs) [reply src/main/resources/mybatis/dynamicStation.xml]: Okay, I changed this . . . not certain if the change was exactly what you had in mind.

Comment (dsteinich) [reply src/main/resources/mybatis/dynamicStation.xml]: Might need the camel cased "periodOfRecord" - to test, output shouldn't include a period of record site without lat/long.

Comment (abriggs-usgs) [reply src/main/resources/mybatis/dynamicStation.xml]: True and fixed.

**Pull request merged**